src/altimeter.cpp

```diff
@@ -38,10 +39,16 @@ void reconnect(LidarLite& lidarLite)
 
 int main(int argc, char **argv) {
 
+
     ros::init(argc, argv, "altimeter");
     ros::NodeHandle n;
     ros::NodeHandle private_nh("~");
 
+    ROS_INFO("altimeter: Attempting to form safety bond");
+    Iarc7Safety::SafetyClient safety_client(n, "Altimeter");
+    ROS_ASSERT_MSG(safety_client.formBond(),
```

Comment:
The bond shouldn't be formed until the node has finished all its startup procedures, i.e. it's connected to the lidarlite successfully.  Might also want to make sure the topics / transforms this depends on are sending messages, but not necessarily.